zilliqa/src/consensus.rs

```diff
@@ -1290,11 +1393,12 @@ impl Consensus {
         Ok(block.committee.leader(view))
     }
 
-    fn check_quorum_in_bits(&self, view: u64, cosigned: &BitSlice) -> Result<()> {
-        let committee = &self
-            .get_block_by_view(view - 1)?
-            .ok_or_else(|| anyhow!("missing block"))?
-            .committee;
+    fn check_quorum_in_bits(
+        &self,
```

Comment:
missing block errors were more common before, now you refer to the parent's committee directly so it is less error prone